index.html

```diff
@@ -0,0 +1,302 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>
+      Digital Identities
+    </title>
+    <script src="https://www.w3.org/Tools/respec/respec-w3c" class=
+    "remove"></script>
+    <script class="remove">
+    "use strict";
+    // See https://github.com/w3c/respec/wiki/ for how to configure ReSpec
+    var respecConfig = {
+      github: "WICG/digital-credentials",
+      editors: [
+        {
+          name: "Marcos Caceres",
+          email: "marcosc@apple.com",
+          company: "Apple Inc.",
+          companyURL: "https://apple.com",
+        },
+        {
+          name: "Sam Goto",
+          email: "goto@google.com",
+          company: "Google Inc.",
+          companyURL: "https://google.com",
+        },
+        // Add additional editors here.
+        // https://github.com/w3c/respec/wiki/editors
+      ],
+      shortName: "digital-identity",
+      specStatus: "CG-DRAFT",
+      group: "wicg",
+      localBiblio: {
+      },
+      xref: {
+        profile: "web-platform",
+      },
+    };
+    </script>
+  </head>
+  <body data-cite="credential-management-1 WebCryptoAPI">
+    <section id="abstract">
+      <p>
+        This document specifies an API to allow user agents to mediate access
+        to, and representing, a digital identity (e.g. a digital driver's
```

Comment:
```suggestion
        to, and representing, a verifiable issued digital identity (e.g. a government issued digital driver's license, a passport issued by a country, a student card or diploma issued by a university, an employee card issued by a company, a passenger's boarding pass issued by an airline, a profile issued by an online social network, membership cards, vaccination records, etc)  
```

Reply:
I think it would be useful to make sure we are all on the same page with regards to how we are expecting this API to be used, so I added a few more concrete examples that I hear.

Does this match your intuition?

Reply:
Whether or not the suggestion above is adopted, it is unclear what `and representing` is connected to.
* Does it mean `to allow user agents to represent a digital identity`?
* Does it mean `an API representing a digital identity`?
* Does it mean something else?

Once this is answered, I will draft a rephrasing ... unless such a rephrasing is how the answer is provided.

----

Separately, I suggest changing `verifiable issued digital identity` to `verifiably-issued digital identity` or `verifiable digital identity`, unless it is really meant to say that an _issued digital identity_ is _verifiable_, which makes me want a clear definition of how an _issued digital identity_ differs from an _**un**issued digital identity_

Reply:
It would be good to include these here now, but eventually move the expanded examples into the yet to be written introduction. 

Also agree with @TallTed, should be "verifiably-issued digital identity". I'll change your suggestion to that. 

@TallTed, can you file a bug instead for us to work on the abstract, intron, and scope?

Reply:
@TallTed filed https://github.com/WICG/digital-credentials/issues/68

Reply:
> Also agree with @TallTed, should be "verifiably-issued digital identity". I'll change your suggestion to that.
"Verifiably-issued digital identity" works for me for now.